### PR TITLE
EZP-30547 Config resolver for SiteAccess

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZP-30547](https://jira.ez.no/browse/EZP-30547)
| Versions      | 2.5, 1.7, 1.13

Tip admonition added on the config resolver for SiteAccess.

### CODE REVIEW

File: docs/cookbook/setting_up_multi_language_siteaccesses.md [hunk 1]
```diff
@@ -144,3 +144,9 @@ php bin/console cache:clear
 ```
 
 You should now be able to load your eZ Platform site in the `en` and `fr` SiteAccess displaying English and French content.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess, use the following command to debug or speed up the configuration process:
```
Comment (andrerom) [inline docs/cookbook/setting_up_multi_language_siteaccesses.md]: In what way do we imply this will speedup the process? What did you have in mind here? :)

Comment (michalmilc90) [reply docs/cookbook/setting_up_multi_language_siteaccesses.md]: I must have misunderstood something 😅 I will remove that fragment 😄

Comment (andrerom) [reply docs/cookbook/setting_up_multi_language_siteaccesses.md]: Could be something like:
`If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:`

File: docs/guide/siteaccess.md [hunk 2]
```diff
@@ -37,6 +37,15 @@ you need to create your own admin SiteAccesses and add them to this group. In ca
...
 
 You configure SiteAccess in your config files (e.g. `ezplatform.yml`) under the `ezpublish.siteacess` keys.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:
+    
+    `bin/console [—-siteaccess=<SA>] ezplatform:debug:config-resolver <param.name>`
+    
+    The command also enables you to test how different parameters work when configuring [scopes](#scope).
```
Comment (andrerom) [inline docs/guide/siteaccess.md]: This is like a tip within a tip, do we have a way to signify that?

Or should we maybe just prefix with something like `Advance Tip:` or `Advance Usage:` ?

Comment (michalmilc90) [reply docs/guide/siteaccess.md]: Will see what can we do about it. ;)

File: docs/cookbook/setting_up_multi_language_siteaccesses.md [hunk 3]
```diff
@@ -144,3 +144,9 @@ php bin/console cache:clear
 ```
 
 You should now be able to load your eZ Platform site in the `en` and `fr` SiteAccess displaying English and French content.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:
```
Comment (MagdalenaZuba) [inline docs/cookbook/setting_up_multi_language_siteaccesses.md]:
```suggestion
    If you encounter issues when configuring SiteAccess or want to check that system uses the correct value, use the following command:
```

File: docs/guide/siteaccess.md [hunk 4]
```diff
@@ -37,6 +37,15 @@ you need to create your own admin SiteAccesses and add them to this group. In ca
 ## Configuring SiteAccesses
 
 You configure SiteAccess in your config files (e.g. `ezplatform.yml`) under the `ezpublish.siteacess` keys.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:
```
Comment (MagdalenaZuba) [inline docs/guide/siteaccess.md]:
```suggestion
    If you encounter issues when configuring SiteAccess or want to check that system uses the correct value, use the following command:
```

**Pull request merged**